src/demo/app.ts

```diff
@@ -2,11 +2,14 @@ import {addSpawnListeners, createApp, createWindow} from 'openfin-service-toolin
 
 import * as ofnotes from '../client/index';
 import {NotificationOptions, NotificationActionEvent, NotificationClosedEvent, NotificationCreatedEvent, create, addEventListener, clear, getAll} from '../client/index';
+import {Events} from '../client/internal';
 
 addSpawnListeners();
 
+const receivedEvents: Events[] = [];
+
 // Mount createWindow and createApp on the window to be used by puppeteer
```

Comment:
Should maybe update this comment since there's a new thing being mounted

Reply:
✅